mangaki/mangaki/management/commands/dedupe.py

```diff
@@ -0,0 +1,113 @@
+from django.core.management.base import BaseCommand, CommandError
+
+import os
+import csv
+from dedupe import Dedupe, consoleLabel, canonicalize
+from mangaki.models import Work
+
+
+def dedupe_training(category):
+    assert (category in ['mangas', 'animes']),"Only mangas or animes needs training"
+    data = {}
+    for work in Work.objects.filter(category__slug=category[:len(category)-1]):
```

Comment:
You can just write `category[:-1]` it'll do the same
(negative indices are rewritten as `len(container) + index`, e.g. `len(container) - 1`).